en/docs/fragments/contributing-connectors.md

```diff
@@ -0,0 +1,15 @@
+
+You are more than welcome to share your connectors to host it in our [connector store](https://store.wso2.com/store/assets/isconnector/list). As an organization that believes in the open source paradigm, we always encourage our community to make contributions so that our community around WSO2 Identity server can get the full benefit of using an open source product.
```

Comment:
```suggestion
You are welcome to share and host your connectors in our [connector store](https://store.wso2.com/store/assets/isconnector/list). As an organization that believes in the open source paradigm, we always encourage our community to make contributions so that our community around WSO2 Identity server can benefit from using an open source product.
```